src/ProgressBar.webmodeler.ts

```diff
@@ -29,11 +29,22 @@ export class preview extends Component<ProgressBarContainerProps, {}> {
             bootstrapStyle: props.bootstrapStyle,
             className: props.class,
             colorSwitch: props.textColorSwitch,
-            maximumValue: 100,
-            progress: 80,
+            displayTextValue: this.getDisplayTextValue(),
+            maximumValue: props.staticMaximumValue,
+            progress: props.staticValue,
             style: ProgressBarContainer.parseStyle(props.style)
         };
     }
+
+    private getDisplayTextValue() {
+        if (this.props.displayText === "attribute") {
+            return `{ ${this.props.displayTextAttribute} }`;
+        } else if (this.props.displayText === "static") {
+            return this.props.displayTextStatic;
+        }
+
+        return "";
+    }
```

Comment:
Missing the `getVisibleProperties` changes like https://github.com/FlockOfBirds/progress-circle/pull/12/files#diff-e8e1d3f637e5239aba496b065ce6c355R53

Reply:
Added